modules/planning/common/reference_line_info.cc

```diff
@@ -64,11 +64,11 @@ bool ReferenceLineInfo::Init(const std::vector<const Obstacle*>& obstacles) {
   Vec2d center(position + vec_to_center.rotate(path_point.theta()));
   Box2d box(center, path_point.theta(), param.length(), param.width());
   // realtime vehicle position
-  Vec2d vehicle_position(vehicle_state_.x(), vehicle_state_.y());
-  Vec2d vehicle_center(vehicle_position +
-                       vec_to_center.rotate(vehicle_state_.heading()));
-  Box2d vehicle_box(vehicle_center, vehicle_state_.heading(), param.length(),
-                    param.width());
+  // Vec2d vehicle_position(vehicle_state_.x(), vehicle_state_.y());
```

Comment:
For commented code, can add a `TODO`, for example in this case, can add `#TODO: uncomment code below to get realtime vehicle position`

Reply:
Right. It is `58  // stitching point` vs `66  // realtime vehicle position`.
But, the `stitching point` should be more `realtime` than the `vehicle_state_ ` who commented as `realtime vehicle position`.
That's why I comment it, not delete. Please double check here.
@jmtao It is better to delete the `fake` `realtime vehicle position` and the following `vehicle_state_` related commented lines. Do you think so?

Reply:
My comment is not about the logic behind the code, but about the code convention. If you want to comment out some code instead of deleting it, leave a `TODO` to explain why comment out these codes.

Reply:
Alright. I have deleted it after double-checking.